utils/src/test/resources/application.conf

```diff
@@ -13,3 +13,7 @@
 
 #system-wide time zone
 timezone="UTC"
+
+defaultTimestampTimeZone="CET"
+defaultTimestampTimeZone-testFormat="Africa/Johannesburg"
```

Comment:
Originally I wanted to use a dot as a delimiter. but as you can judge from my CQC question, it didn't really work.  I hope this is acceptable.

Reply:
Camel case seems weird for me in conf. How about `default_timestamp_timezone` and `default_timestamp_timezone_test_format`?

Reply:
> 
> 
> My first choice would probably be something like
> 
> ```
> defaultTimestampTimeZone {
>   defaultFormat : "CET",
>   testFormat: "Africa/Johannesburg"
> }
> ```
> 
> to avoid the "magic" of the loading where the `-{format}` suffix may be present. But I see your point that this implementation's important upside is that it is backwards-compatible.
> 
> With my suggested solution, there would have to be some fallbacking on the config reading - if an object is found, use new processing, if just value, fallback it to `defaultTimestampTimeZone { defaultFormat : "$valueHere" }`.
> 
> So your solution seems acceptable to me as-is.

I wouldn't mind some extra code to ensure backward compatibility while enhancing to the object model nicely. But what you wrote applies only for json formatted config files, doesn't it? Our config are the classical `key=value` type of file.

Reply:
I think in HOCON, `:` and `=` are interchangeable. https://github.com/lightbend/config#features-of-hocon